user-dao-intro.adoc

```diff
@@ -0,0 +1,144 @@
+
+It achieves this by introducing a unit of value called the **BSQ token** that enables Bisq stakeholders—contributors, traders, or anyone who owns BSQ—to make subjective value judgments. This token underlies all actions in the DAO.
+
+=== BSQ Token
+
+A BSQ token is a **colored coin** on the Bitcoin blockchain. More plainly, a BSQ token is merely a few satoshis with some additional properties that identify it as BSQ in Bisq software. Outside Bisq, a BSQ token just looks like a few satoshis. But on Bisq, the additional properties have the following results:
+
+1. the satoshis take on the market value of a BSQ token
+2. the owner of the satoshis is endowed with power to participate in various Bisq governance functions
+
+See it for yourself: here's a BSQ token on an ordinary block explorer, and here's that same UTXO on a BSQ block explorer.
+
+The value of a BSQ token is determined by the market as stakeholders buy and sell tokens to perform various functions in the DAO. Because BSQ tokens are only recognized by Bisq software, BSQ trading can only happen in Bisq software.
+
+NOTE: The BSQ token is not associated with any kind of initial coin offering (ICO) or capital-raising initiative. It's merely a tool to decentralize the governance of the already-functional Bisq exchange service. As of December 2018, Bisq has facilitated over 19,000 trades without major incident since it went into production in April 2016.
+
+You can learn more about colored coins https://www.youtube.com/watch?v=68_DU1c0Cac[here in this interview^]. Technical details of BSQ tokens are available <<dao/specification#,here>>.
+
+=== Overview
+
+Here's a graphical overview of how the DAO works at a high level:
+
+image::user-dao-diagram.png[Overview of the Bisq DAO]
+
+Let's observe some of the dynamics of this system:
+
+* Contributors _maintain_ Bisq through valuable work
+* Traders _use_ Bisq to buy and sell bitcoin and other cryptocurrencies
+* Contributors _earn_ BSQ tokens when their compensation requests are approved through voting
+* Traders _buy_ BSQ tokens in order to pay lower trading fees
+* Stakeholders _vote_ on compensation requests
+* Contributors can _lock_ a bond in BSQ to ensure integrity in high-trust roles
+
+Now let's see how these dynamics enable the Bisq DAO to provide 3 core governance functions without any central points of authority.
+
+=== Earn and distribute revenue
+
+Every project needs to be financial sustainable to fund ongoing operations and development. But traditional means of handling revenue and revenue distribution are necessarily centralized—any receiving address or account must be owned by one person, or a small group; determining how much a person should be paid must be determined by one person, or a small group; etc.
+
+The Bisq DAO seeks to avoid such centralization of authority. So, instead of taking in trading fees and then distributing them, the Bisq DAO distributes fees first and then puts them up for sale.
+
```

Comment:
I think that is not clear: "the Bisq DAO distributes fees first and then puts them up for sale."
Maybe: 
So, instead of having a central authority taking in trading fees and then distributing them to contributors, the fees get burned or "uncolored" and by that each stakeholder experience a slight value increase because BSQ is taken out of supply thus making each unit of a token slightly more valuable.

Reply:
I didn't implement this text exactly, but I did reword the section and included a link to the Phase Zero section on monetary policy. 

Can include the bit on burning BSQ increasing the value of existing BSQ tokens if you want it, but I didn't want to include that without acknowledging the reverse effect too.